### DISCUSSION TITLE
Copilot cannot login: Cannot initialize the Github login progress

### DISCUSSION THREAD

**Discussion by tsavpyn** [community/community#17429] [Copilot Conversations]
I tried all versions of copilot plugin but non of them works. 
In some newer version, it says Cannot initialize the github login progress. 
In older versions under 1.19, I logged in successfully but it always says 'No completion found'. Sometimes I tried to logout and login again but it has been waiting to authorize all the time. 

The client I use is PyCharm Professional 2022.1.1, I never use proxy. The system environment is Windows 7 sp2.

**Comment by LuanJining** [comment]
bro, i have the same problem

**Reply by tsavpyn** [reply]
Do you know how to fix this?

**Reply by tsavpyn** [reply]
Is that blocked in China?

**Comment by davecheney** [comment]
@etoGrassie your client is likely out of date, please upgrade. https://github.com/github/feedback/discussions/16815

**Reply by tsavpyn** [reply]
Thx for ur answer. But the client I use is PyCharm Professional 2022.1.1. It is the latest version.

**Reply by tsavpyn** [reply]
And my github copilot plugin was downloaded from the plugin market, the version was 1.23.xxxx

**Comment by jansorg** [comment]
If you're using a proxy server, please refer to https://github.com/github/feedback/discussions/16230#discussioncomment-2747034

**Reply by tsavpyn** [reply]
Thx for ur answer. But I never use proxy.

**Comment by rafavolpato** [comment]
Same problem in pycharm.

**Comment by tsavpyn** [comment]
This problem is solved now. The answer is, Copilot does not support Windows 7 and older systems. 
I have tested Copilot on different systems, Windows 7 and older systems do not have winsock3 (maybe this name or other I forgot) dll do it doesn't work.

**Reply by rafavolpato** [reply]
In fact I'm using Linux:
Description:	Ubuntu 20.04.3 LTS

I install the plugin. Restart the IDE (pycharm). Copy and open the browser, paste the code and it says that the login is complete.
When I go back to pycharm, it says that the configuration is not ready.
I tried to reinstall the plugin and also tried to restart my computer.
But didn't work.

**Reply by kaiwenShen** [reply]
same

**Comment by tsavpyn** [comment]
That is really weird. But I am so sorry that I know nothing to Linux (*laugh). I recommend you to check your internet settings and make sure that you are not using VPN or proxy. If there is a firewall or something like firewall, turning it off maybe helpful. And upgrading system maybe a good choice.

**Comment by rafavolpato** [comment]
Hi,

I also saw a comment about proxy.
But I was not using VPN or proxy and there is no firewall.


Regards,
Rafael Cunha
+55 (61) 98107-0362

**Comment by wizard-waqas** [comment]
The reason why it wasn't working for me was because I didn't signup for copilot in the first place. 

Sign up here: https://github.com/features/copilot

**Comment by leonardomorenozaga** [comment]
I got stuck also, the signup prompt was no presented to do the auth.
Disabling, reloading, reenabling the extension did the trick to kick again the authentication prompt.
